Modules/Common/BeaconChainLikeAttestationsModule.php

```diff
@@ -13,8 +13,8 @@ abstract class BeaconChainLikeAttestationsModule extends CoreModule
 
     public ?BlockHashFormat $block_hash_format = BlockHashFormat::HexWithout0x;
     public ?AddressFormat $address_format = AddressFormat::AlphaNumeric;
-    public ?TransactionHashFormat $transaction_hash_format = TransactionHashFormat::None;
-    public ?TransactionRenderModel $transaction_render_model = TransactionRenderModel::None;
+    public ?TransactionHashFormat $transaction_hash_format = TransactionHashFormat::AlphaNumeric;
+    public ?TransactionRenderModel $transaction_render_model = TransactionRenderModel::Mixed;
```

Comment:
This should be reverted, see https://github.com/3xplcom/Core/commit/7c879b17334f1ad68fdec875c1edc10d6fe7582d

Reply:
Can u please help me I only just seen this messsge sorry

Reply:
And how do I do that sorry